.eslintrc

```diff
@@ -5,7 +5,8 @@
         "es6": true
     },
     "ecmaFeatures": {
-        "modules": true
+        "modules": true,
+        "experimentalObjectRestSpread": true,
```

Comment:
This tells ESlint to expect use of the spread operator so my editor doesn't gripe at me :smile: 
 * example - `myFunction(...iterableObj)`
 * Something to look into later might be using [babel-eslint](https://www.npmjs.com/package/babel-eslint) as a parser. It tends to be more compatible with ES6 & 7 I've found.